### DISCUSSION TITLE
1.5.4

### DISCUSSION THREAD

**Discussion by bellejt** [fedorovvl/tso_client#147] [General]
not working.Gives mail error .Maybe we need to set the - skip again ?

**Comment by fedorovvl** [comment]
don't set "upstream swf" checkbox if u not on frensh region

**Reply by bellejt** [reply]
how ?

**Reply by fedorovvl** [reply]
![изображение](https://user-images.githubusercontent.com/31754742/203348876-6a5633bd-dfad-4b20-88bc-f094454f1c5f.png)
easily =)

**Reply by bellejt** [reply]
THX a lot.did the trick :)

**Reply by Anprivate** [reply]
I'm in the Turkiye now, playing "European english" and game starts only with "upstream swf" checkbox checked. Just for information :)

**Comment by igorSof** [comment]
если запускаю клиент с ключом debug, то вызов скриптов F3 F4 одно сплошное мучение, игра подвисает крутит но всё делает, ждать по минуте приходится наверное, в итоге пока ключ убрал

**Reply by Tartapuce** [reply]
[](url)v1.5.4 client not working
I update client, and client.swf, but i get a message tu clean cache an never can connect....
![Client obsolete](https://user-images.githubusercontent.com/118884539/203497892-95e9dcf8-9e66-42fa-b42f-da95633829a6.jpg)
What can i do ?

**Reply by skelgaard** [reply]
there is maintenence and server is down

**Reply by Tartapuce** [reply]
Sorry, i don't think, i can connect on unity...
The last time i get this error, the client.swf wasn't correct and after been corrected, all goes on !

OK, sorry, i'm on french region and set "upstream swf" checkbox. I set it, and it works....

**Reply by BasilMas** [reply]
> если запускаю клиент с ключом debug, то вызов скриптов F3 F4 одно сплошное мучение, игра подвисает крутит но всё делает, ждать по минуте приходится наверное, в итоге пока ключ убрал

А зачем ты запускаешь с ключом дебаг? какой смысл в этом?

**Reply by igorSof** [reply]
можно команды/скрипты выполнять прямо налету, набрал и сразу выполнил, для отлавливания ошибок.

**Reply by fedorovvl** [reply]
нажмите customcode, потом esc и потом ф12
откроется окно дебагера.. лагать перестанет
